docs/ref/clvm.md

```diff
@@ -125,24 +125,23 @@ And in this example, an atom that was read in as an integer is appended to a str
 
 ### Atoms as Byte Arrays
 
-The atom is treated as an array of bytes, with a length. No specific semantics are assumed, except as specified in the instruction.
+The atom is an array of bytes, with a length. No specific semantics are assumed, except as specified in the instruction.
```

Comment:
the atom *is* a byte array. it's only treated as different things by operands, but that doesn't change what it *is*.